content/terraform-docs-common/docs/cloud-docs/api-docs/hold-your-own-key/azure-oidc-configurations.mdx

```diff
@@ -0,0 +1,170 @@
+---
+page_title: Azure specific /oidc-configuration API reference for HCP Terraform
+description: >-
+ Use the HCP Terraform API's `/oidc-configuration` endpoint to create and view your Azure OIDC configurations for Hold Your Own Key (HYOK) in HCP Terraform.
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+
+[201]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
+
+[202]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
+
+[204]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204
+
+[400]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
+
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+
+[409]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
+
+[412]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412
+
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+
+[429]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429
+
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+
+[JSON API document]: /terraform/cloud-docs/api-docs#json-api-documents
+
+[JSON API error object]: https://jsonapi.org/format/#error-objects
+
+# Azure OIDC Configuration API reference
+
+## Create OIDC Configuration
+ `POST /api/v2/organizations/:organization_id/oidc-configurations`
+
+ | Parameter           | Description                  |
+ | ------------------- | ---------------------------- |
+ | `:organization_id` | The ID of your organization. |
+
+ | Status  | Response                  | Reason                                                            |
+ |---------|---------------------------|-------------------------------------------------------------------|
+ | [201][] | [JSON API document][]     | Successfully created OIDC configuration.  |
+ | [404][] | [JSON API error object][] | Organization not found, or user unauthorized to perform action.    |
+
+### Request Body
+
+| Key path                         | Type    | Default   | Description                                                                                                                                                                                               |
+| -------------------------------- | ------- | --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `data.type`                      | string  |           | Must be `"azure-oidc-configurations"`.                                                                                                                                                                               |
```

Comment:
I think it would be worth including the required attributes here.
```suggestion
| `data.type`                      | string  |           | Must be `"azure-oidc-configurations"`. |
| `data.attributes.client-id`                      | string  |           | The Client (or Application) ID of your Entra ID application. |
| `data.attributes.subscription-id`           | string  |          | The ID of your Azure subscription. |
| `data.attributes.client-id`                      | string  |           | The Tenant (or Directory) ID of your Entra ID application. |
```